ecr/fetcher_test.go

```diff
@@ -133,10 +133,10 @@ func TestFetchManifest(t *testing.T) {
 				assert.Equal(t, repository, aws.StringValue(input.RepositoryName))
 				assert.Equal(t, []*ecr.ImageIdentifier{{ImageTag: aws.String(imageTag)}}, input.ImageIds)
 				// TODO: Determine if we should be matching the requested media type from containerd
-				assert.Equal(t, []*string{
-					aws.String(ocispec.MediaTypeImageManifest),
-					aws.String(images.MediaTypeDockerSchema2Manifest),
-				}, input.AcceptedMediaTypes)
+				assert.ElementsMatch(t, []string{
+					ocispec.MediaTypeImageManifest,
+					images.MediaTypeDockerSchema2Manifest,
+				}, aws.StringValueSlice(input.AcceptedMediaTypes))
```

Comment:
Order can matter here; the previously asserted order prefers `ocispec.MediaTypeImageManifest`.

(This is a hint for conversion; if you have content that does not match either media type but can be converted to either, we prefer `ocispec.MediaTypeImagemanifest`.)

Reply:
`assert.ElementsMatch` does not assert ordering.  Can you make sure that we are asserting ordering?